bot/persistence/resources/init_db.sql

```diff
@@ -5,6 +5,14 @@ CREATE TABLE IF NOT EXISTS Modmail(
     Timestamp TEXT NOT NULL
 );
 
+CREATE TABLE IF NOT EXISTS Suggestion(
+    ID INTEGER PRIMARY KEY,
+    MessageID TEXT,
+    AuthorID TEXT NOT NULL,
+    StatusID SMALLINT NOT NULL DEFAULT 0,
+    Timestamp TEXT NOT NULL
```

Comment:
If we don't need this, remove it. Also we have the timestamp on the message.

Reply:
It would be possible that I need this timestamp in the future. I'm considering if I should implement a commands which returns me a list of all suggestions which have no decision or which are currently being considered. When I do that, I would like to have the timestampt to see since when the suggestion is "open" or something. Of course I could use the timestamp from the message itself but I tried that with the modmail before and the API calls really slowed things down. That's why I store the timestamp for the modmail in the db too.